fed_test_utils/fed.go

```diff
@@ -153,51 +154,89 @@ func NewFedTest(t *testing.T, originConfig string) (ft *FedTest) {
 	directorStartTime := time.Now().Add(-6 * time.Minute)
 	director.SetStartupTime(directorStartTime)
 
-	config.InitConfig()
 	err = config.InitServer(ctx, modules)
 	require.NoError(t, err)
 
 	// Read in any config we may have set
+	var originConf interface{}
 	if originConfig != "" {
 		viper.SetConfigType("yaml")
 		err = viper.MergeConfig(strings.NewReader(originConfig))
 		require.NoError(t, err, "error reading config")
+
+		err := yaml.Unmarshal([]byte(originConfig), &originConf)
+		require.NoError(t, err, "error unmarshalling into interface")
+
+		confMap := originConf.(map[string]interface{})
+
+		originRaw, exists := confMap["Origin"]
+
+		if exists {
```

Comment:
This might be a bit cleaner as:
```
if originRaw, exists := confMap["Origin"]; exists {
    // blah
}
```

since `originRaw` is not used outside the scope of this condition.

I think it might also be a fatal error if we don't pass this existence check -- that would indicate the test thinks it was provided origin configuration, but that configuration doesn't contain `Origin:`.